spec/classes/snmp_init_spec.rb

```diff
@@ -745,7 +745,7 @@
             end
 
             it { is_expected.to contain_service('snmpd').with_ensure('running') }
-            it { is_expected.not_to contain_service('snmptrapd') }
+            it { is_expected.to contain_service('snmptrapd') }
```

Comment:
Just to be sure: this is a fix for a broken test which is not part of CI so we did not catch?

Reply:
> Just to be sure: this is a fix for a broken test which is not part of CI so we did not catch?

Actually I think that line is wrong, and that whole block would never be run anyway. Pushed a commit to remove it.